deployability/modules/workflow_engine/examples/dtt1-agents-poc.yaml

```diff
@@ -0,0 +1,170 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+version: 0.1
+description: This workflow is used to test agents deployment por DDT1 PoC
```

Comment:
We should remember to remove the DTT1 PoC references, also, it is a typo, its DTT not DDT

Reply:
Done in [6941f08](https://github.com/wazuh/wazuh-qa/pull/4990/commits/6941f0899b9f8fea932167fe3aa91d484efa29c4)